handlers/authorization/check_student_existance.py

```diff
@@ -31,11 +33,11 @@ async def check_student(student: StudentCheckExistanceIn):
     token = hashlib.sha1(os.urandom(128)).hexdigest()
     expires = datetime.utcnow() + timedelta(seconds=Settings.TOKEN_LIFE_TIME)
 
-    query = one_time_token.insert().values(student_id=student_id, token=token,
-                                           expires=expires).returning(one_time_token.c.token_id)                      
+    query = insert(OneTimeToken).values(student_id=student_id, token=token,
+                                        expires=expires).returning(OneTimeToken.token_id)
     last_record_id = await database.execute(query)
 
-    query = one_time_token.select().where(one_time_token.c.token_id == last_record_id)
+    query = select(OneTimeToken).where(OneTimeToken.token_id == last_record_id)
```

Comment:
No need to make an extra select request, you can combine insert + returning, and also with scalar_one()

Reply:
You mean, I should do from this:
```
    query = insert(OneTimeToken).values(student_id=student_id, token=token,
                                        expires=expires).returning(OneTimeToken.token_id)
    last_record_id = await database.execute(query)
```
this?
```
    last_record_id = await database.execute(insert(OneTimeToken).values(student_id=student_id, token=token,
                                        expires=expires).returning(OneTimeToken.token_id))

```

Reply:
Issue will be resolved in future tickets